### PR TITLE
Expand discussion about timing vs external pressure

### PR DESCRIPTION
**Pull request by kenchris**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,9 +66,13 @@ These popular [real-time applications](https://en.wikipedia.org/wiki/Real-time_c
 are classified as _soft_. That is, the quality of service degrades if the system is exercised beyond certain states, but does not lead to a total system failure.
 These _soft_ real-time applications greatly benefit from being able to adapt their workloads based on CPU consumption/pressure.
 
-Web apps can also suffer from high CPU pressure beyond the apps control, which can result in a degraded interactivity experience. This can impact the time it takes
-for complex components to render and thus increase the response time to interactions, resulting in a degraded user experience. This can be mitigated by rendering
-simpler content or skeleton content in cases where the CPU pressure is high.
+It the use-cases is to adopt the user experience to the user system at hand, measuring the time to archieve
```
Comment (kenchris) [inline README.md:69]: If

File: README.md [hunk 2]
```diff
@@ -66,10 +66,10 @@ These popular [real-time applications](https://en.wikipedia.org/wiki/Real-time_c
 are classified as _soft_. That is, the quality of service degrades if the system is exercised beyond certain states, but does not lead to a total system failure.
 These _soft_ real-time applications greatly benefit from being able to adapt their workloads based on CPU consumption/pressure.
 
-It the use-cases is to adopt the user experience to the user system at hand, measuring the time to archieve
-certain tasks is an option, but web apps can also suffer from unusual high CPU pressure beyond the apps control.
+If the use-cases is to adopt the user experience to the user system at hand, measuring the time to archieve
```
Comment (arskama) [inline README.md:69]: "are to adapt"

Comment (arskama) [inline README.md:69]: time to achieve

Comment (kenchris) [reply README.md]: fixed

Comment (kenchris) [reply README.md]: fixed

**Review by arskama — COMMENTED** [review]
LGTM

**Pull request merged**